matminer/featurizers/composition.py

```diff
@@ -254,6 +255,83 @@ def citations(self):
     def implementors(self):
         return ["Jiming Chen", "Logan Ward", "Anubhav Jain", "Alex Dunn"]
 
+class Meredig(BaseFeaturizer):
+    """
+    Class to calculate features as defined in Meredig et. al.
```

Comment:
We should expand on the documentation here, following the style described in the link Alex pointed out. E.g., what are the names of the features generated by this class.

Reply:
I expanded the documentation in the latest commit, let me know if it looks good